helm/values.yaml

```diff
@@ -71,38 +84,26 @@ env:
   metrics:
     enabled: false
     url: http://localhost:55681/v1/metrics
+  queue:
+    jobManagerBaseUrl: "http://localhost:8081"
+    heartbeat:
+      heartbeatManagerBaseUrl: "http://localhost:8089"
+      heartbeatIntervalMs: 3000
+    dequeueIntervalMs: 1000
+    jobType: "TilesSeeding"
+    tilesTaskType: "TilesSeeding"
+  mapproxy:
+    mapproxyApiUrl: "http://localhost:8083"
+  seedAttempts: 5
+  seedConcurrency: 5
```

Comment:
where is the storage type defintion is set? (FS or S3)

Reply:
it's not controlled by the service, that storage type is related to the layer's source as defined on mapproxy.yaml,
so the cache-seeder only reads the tile source as provided on the configuration and executes seeding.
In theory its works, but if we would like it, so the deployment should support specific mount for tiles location (if the source is FS)

Reply:
after talking with you.
agreed and added FS full support